src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -483,14 +523,18 @@ ReactDOMComponent.Mixin = {
    * @param {ReactReconcileTransaction|ReactServerRenderingTransaction} transaction
    * @param {?ReactDOMComponent} the containing DOM component instance
    * @param {?object} info about the native container
-   * @param {object} context
+	 * @param {object} context
+	 * @param {?DOMNode|Array<DOMNode>} when reconnecting to server markup, the
+   *   DOM node to reuse or an array of DOM nodes to reuse. Note that with this
```

Comment:
"array of DOM nodes to reuse.".  As per previous discussion, let's fix up the wording here to be a little more clear/correct.